Example in Angular

guzbarraf opened the pull request:
Example of using Final Space API in Angular

guzbarraf commented:
**A screenshot of the example code in Angular**

![Captura de Pantalla 2020-10-27 a la(s) 22 36 24](https://user-images.githubusercontent.com/5010387/97391492-ede76480-18a4-11eb-8787-430b1ed34d1f.png)

guzbarraf commented:
Done @lelouchB

lelouchB commented:
> 
> 
> Done @lelouchB

That was superfast @guzbarraf

lelouchB commented on frontend/docs/angular.mdx:
```diff
@@ -0,0 +1,107 @@
+  <div *ngFor="let item of items" class="card">
+    <div class="card shadow--tl">
+      <div class="card__image">
+        <img
```
@guzbarraf can you resolve this issue?

guzbarraf commented:
> That was superfast @guzbarraf

It wasn't quick, I got to work, without realizing that I hadn't asked for the issue

lelouchB commented on frontend/docs/angular.mdx:
```diff
@@ -0,0 +1,107 @@
+---
+
+
+<Tabs
```
We don't need this tabs.

Since there is only tab
Can you remove this ?

guzbarraf replied:
ok, i remove it

lelouchB commented on frontend/docs/angular.mdx:
```diff
@@ -0,0 +1,107 @@
+  }
+}
+
+{/* app.component.html*/}
```
Can you separate it into two code snippets 
```
{/* app.component.ts*/}
```

```
{/* app.component.html*/}
```

Also for css

guzbarraf replied:
ok

guzbarraf commented:
@lelouchB 
If the PR is approved,
does the project need an example of Angular with Axios?

lelouchB reviewed (commented):
> 
> 
> @lelouchB
> If the PR is approved,
> does the project need an example of Angular with Axios?

You can add one in the tabs like done with Vue+Axios example, if you want.

lelouchB commented:
One more thing @guzbarraf 
I was trying to make a project of this example on StackBlitz and CodeSandbox,  but I am a completely stranger to Angular so can you please copy this example to CodeSandbox or StackBltiz so I can fork it and add the link in the example page.


Thanks

guzbarraf commented:
Hi @lelouchB 

I have updated the PR with the mentioned changes,
removed the tabs,
adding the code snippets
and the example in CodeSandbox for the live example,
but I don't understand the error that Github's Labeler throws,
can you help me with that?

guzbarraf reviewed (commented):
> One more thing @guzbarraf
> I was trying to make a project of this example on StackBlitz and CodeSandbox, but I am a completely stranger to Angular so can you please copy this example to CodeSandbox or StackBltiz so I can fork it and add the link in the example page.
> 
> Thanks

Done in CodeSandbox in this url [https://codesandbox.io/s/final-space-api-with-angular-1mvg2](url)

lelouchB commented:
Hey @guzbarraf 
Awesome 

Dont worry about labeler, it was just something I was trying yesterday.
Seems like it is causing unnecessary errors.  I will remove it.

I will review and merge this PR  some time today.

Thanks

guzbarraf commented:
Hi @lelouchB
can you still add hacktoberfest labels to these PRs,
so that they count in the hacktoberfest?
